pkg/planner/cardinality/selectivity.go

```diff
@@ -621,6 +671,34 @@ func getMaskAndRanges(ctx sessionctx.Context, exprs []expression.Expression, ran
 	return mask, ranges, false, nil
 }
 
+func getMaskAndSelectivityForMVIndex(
+	ctx sessionctx.Context,
+	coll *statistics.HistColl,
+	id int64,
+	remainedExprs []expression.Expression,
+) (float64, int64, bool) {
+	cols := coll.MVIdx2Columns[id]
+	if len(cols) == 0 {
+		return 1.0, 0, false
+	}
+	accessConds, _ := CollectFilters4MVIndex(ctx, remainedExprs, cols)
```

Comment:
Should we multiply the selectivity with 0.8 if there is any remaining filter?

Reply:
No. This is only for finding the access conditions. If some conditions don't become access conditions, they are recognized as not covered, and processed by the existing logic in `Selectivity()` by * 0.8.